### PR TITLE
sourceMaps

### PR DESCRIPTION
**Pull request by jalet**
Create source maps for production builds. This is just to make errors from Sentry easier to debug.

### CODE REVIEW

**Comment by jalet** [comment]
Ping @janneh @marreman

File: programs/compile/webpack.config.js [hunk 1]
```diff
@@ -89,9 +89,11 @@ const createRules = (args, opts) => ({
 })
 
 function createWebpackConfig(args = [], opts = {}) {
+  const isProduction = () => args.includes("--production")
+  const useDevServer = () => args.includes("--dev-server")
```
Comment (marreman) [inline programs/compile/webpack.config.js]: These don't need to be functions I guess?

Comment (jalet) [inline programs/compile/webpack.config.js]: doooh!

Comment (emiloberg) [inline programs/compile/webpack.config.js]: 1johrf

**Pull request merged**